src/fides/api/models/detection_discovery.py

```diff
@@ -352,7 +380,6 @@ class MonitorExecution(Base):
     configuration details used in connecting to the external data store.
     """
 
-    id = Column(String, primary_key=True)
```

Comment:
What's happening here? We're removing the `id` property of `MonitorExecution`?

Reply:
so my intention here was to clean up from something we missed when creating the `MonitorExecution` model class. instead of defining our own `id` field/column in the `MonitorExecution` class, we should have relied on the `id` field/column that's defined in the `FidesBase` ORM class that we inherit from. this provides some 'builtin' support for randomized ID generation, and promotes consistency.

so i'm trying to get us back to that better standard here. that being said, i need to be a bit careful i don't break anything on deployments that have already upgraded and have some `MonitorExecution` records with the 'old' `id` definition. i'll do some testing on that before we push this through...

but does that make sense?

Reply:
Makes sense, I'll definitely be on the lookout for that in the future